rest_tools/server/arghandler.py

```diff
@@ -18,11 +17,9 @@ class _NoDefaultValue:  # pylint: disable=R0903
 NO_DEFAULT = _NoDefaultValue()
 
 
-def _get_json_body_arguments(
-    request_handler: tornado.web.RequestHandler,
-) -> Dict[str, Any]:
+def _parse_json_body_arguments(request_body: bytes,) -> Dict[str, Any]:
```

Comment:
The comma after `bytes` here looks weird to my eyes when not followed by another parameter.

Reply:
yup, the black formatter does that sometimes when multi-liners collapse to one-liners